scripts/vnet_route_check.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import json
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicV2Connector
+
+
+def get_vnet_intfs():
+    config_db = ConfigDBConnector()
+    config_db.connect('CONFIG_DB')
+
+    intfs_data = config_db.get_table("INTERFACE")
+    vlan_intfs_data = config_db.get_table("VLAN_INTERFACE")
+
+    vnet_intfs = {}
+    for k, v in intfs_data.items():
+        if 'vnet_name' in v:
+            vnet_name = v['vnet_name']
+            if vnet_name in vnet_intfs:
+                vnet_intfs[vnet_name].append(k)
+            else:
+                vnet_intfs[vnet_name] = [k]
+
+    for k, v in vlan_intfs_data.items():
+        if 'vnet_name' in v:
+            vnet_name = v['vnet_name']
+            if vnet_name in vnet_intfs:
+                vnet_intfs[vnet_name].append(k)
+            else:
+                vnet_intfs[vnet_name] = [k]
+
+    return vnet_intfs
+
+
+def get_rifs_oid():
+    db = SonicV2Connector(host='127.0.0.1')
+    db.connect(db.COUNTERS_DB)
+
+    keys = db.get_all(db.COUNTERS_DB, 'COUNTERS_RIF_NAME_MAP');
+
+    return keys
+
+
+def get_vnet_rifs_oid():
+    intfs = get_vnet_intfs()
+    intfs_oids = get_rifs_oid()
+
+    list = [intfs[k] for k in intfs]
+    flattened = [val for sublist in list for val in sublist]
+
+    oids = {}
+
+    for i in intfs_oids:
+        if i in flattened:
+            oids[i] = intfs_oids[i]
+
+    return oids
+
+
+def get_vrf_entries():
```

Comment:
should you add from_asic_db so it will be clear from where the information will be provided to the calling function?